sdk/remoterendering/mixedreality-remoterendering/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2020 Microsoft
```

Comment:
I think it is worth regenerating using the latest code gen (just do not specify a version) to update the license year and get some of the recent improvements/fixes as well.

Reply:
By "code gen" I assume you mean autorest, or is there another process I should use? Autorest doesn't seem to touch this file.

I note that the other SDKs are also at 2020 at least on my branch (which I rebased last week).

Reply:
Sorry for the confusion, yes I meant autorest. For packages that has 2020 in their license, they were last regenerated and/or released in 2020. Once they get regenerated for their next release, their license has to have the correct release year.

Reply:
@deyaaeldeen Apologies, but I'm still confused. I tried running autorest with no arguments, and it only touches files inside the generated folder, AFAICT. I can manually update the year here, but I don't think that's what you are suggesting.

Reply:
Ok I see your point now. The license will not get updated unless you ask autorest to generate metadata and we do not do that after the very first generation so we do not overwrite hand-written changes to readme, package.json, etc. I also checked other packages and all of them has this issue so it is not unique for yours. Feel free to ignore it for now and I will talk to the team about it.